website/docs/guidelines/pull-request/acceptance-criteria.md

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_label: Contribution Acceptance Criteria
+sidebar_position: 3
+---
+
+# Contribution Acceptance Criteria
+
+To ensure that a PR can be approved, the following criteria must be met:
```

Comment:
```suggestion
To ensure that a PR can be approved, the following criteria are met:
```

Reply:
What do you think if we change that sentence with: `To increase the likelihood of PR approval, consider meeting the following criteria:`